x-pack/plugins/security_solution/public/detections/components/rules/description_step/index.tsx

```diff
@@ -106,7 +117,13 @@ export const StepRuleDescriptionComponent = <T,>({
 
     return [
       ...acc,
-      ...buildListItems(data, pick(key, schema), filterManager, license, indexPatterns),
+      ...buildListItems(
+        data,
+        pick(key, schema),
+        filterManager,
+        license,
+        dataViewInstance as DataViewBase
```

Comment:
is this possible to avoid type castings?

Reply:
Kind of the same situation as above except this is due to the unified search team's component `FilterBadgeGroup` needing a `DataViewBase` type to be passed to it, which the `dataViewInstance` is compatible with. Until that changes, we'll need the casting unfortunately.